src/seedu/addressbook/data/tagging/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tagging;
+
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.*;
+
+public class Tagging {
```

Comment:
Lacking a header comment here. An example could be : Represents an adding or removing a Tag to/from a Person